py3status/modules/arch_updates.py

```diff
@@ -8,11 +8,14 @@
 
 Configuration parameters:
     cache_timeout: How often we refresh this module in seconds (default 600)
+    check_aur_cmd: Command to run to check AUR updates,
+        must print one line per update on STDOUT
+        (default 'cower -u')
```

Comment:
(Imo) I'm biased toward `arch_update: module from scratch`, I acknowledge `aur` is popular term word and also a well familiar repository. Users still can use other repositories too (unofficial, not_trusted, local, or just foreign). So I think maybe to stay generic, we would/could avoid the `aur` name... eg `foreign_command = 'cower -u'`. Some might not like this name. Otherwise, maybe `aur_command` is better.

Also, having this new config makes `include_aur` useless as users could use `foreign_command = None` or `aur_command = ''` instead.